src/js/blight/BlightContainer.jsx

```diff
@@ -5,18 +5,47 @@ export default class BlightContainer extends Component {
     super(props);
     this.totalBlight = this.props.players * 2;
 
-    this.state = { numBlight: this.totalBlight };
+    this.state = { 
+      numBlight: this.totalBlight,
+      blightCardName: "No Effects",
+      blightCardEffect: "",
+      endGameTrigger: 0
+    };
 
     this.handleAddBlight = this.handleAddBlight.bind(this);
     this.handleRemoveBlight = this.handleRemoveBlight.bind(this);
+    this.renderBlightCard = this.renderBlightCard.bind(this);
 
   }
+  renderBlightCard(blights) {
```

Comment:
This doesn't really render a blight card, it is just handling the logic when you remove a blight. Maybe it's best for this to go with `handleRemoveBlight`